doc/nrf/drivers/wifi/nrf70_native.rst

```diff
@@ -118,3 +118,31 @@ To configure 15 dBm, use the value ``15``.
    };
 
 See the DTS binding documentation for more information.
+
+TX power calculation
+********************
+
+The driver will always attempt to set the TX power to the maximum supported level, while ensuring the following constraints:
+
+* EVM and SEM for the modulation type/datarate are satisfied (IEEE 802.11 requirement).
+* In band regulatory power limits are satisfied (FCC / CE requirements).
+* Out of band regulatory power limits are satisfied (FCC / CE requirements).
+
+TX power is calculated as follows:
+
+TX Power = min ((P\ :sub:`reg`\  - ``ANT_GAIN``), P\ :sub:`max`\) - ``EDGE_BACKOFF``
+
+P\ :sub:`max`\  = min (P\ :sub:`ps`\  , P\ :sub:`max-tx-pwr`\)
+
+   where the following parameters are used:
+
+   * P\ :sub:`reg`\  is the applicable regulatory power limit, described here :ref:`ug_nrf70_developing_regulatory_support`.
+   * ``ANT_GAIN`` is the compensation for the antenna gain in the TX direction, described here `Antenna gain compensation`_.
+   * ``EDGE_BACKOFF`` is the backoff applied to band edge channels, described here `Band edge compensation`_.
+   * P\ :sub:`ps`\  is the maximum power level for the package type, modulation and band as described here `Electrical specification for nRF7002`_.
+   * P\ :sub:`max-tx-pwr`\  is the sub-band power limit, dependent on the PCB design.
+
+.. note::
+
+   The behavior described above applies when using the system driver or radio test with default TX power.
+   (i.e. tx_power command not issued).
```

Comment:
This is a bit misleading that these rules aren't applied to custom TX powers, does that mean for custom TX powers no regulatory rules are enforces?